src/lua/digest.lua

```diff
@@ -149,13 +149,6 @@ setmetatable(PMurHash, {
 
 local CRC32
 local CRC32_methods = {
-    update = function(self, str)
```

Comment:
I guess this line could be removed as well:
https://github.com/tarantool/tarantool/blob/86a2edab6df17a3b0e5ef61ef52a244b33206e51/src/lua/digest.lua#L20

Reply:
No, it could not. Please, see its usage in [digest.crc32_update](https://github.com/tarantool/tarantool/blob/b2c3f868e98f5ff35469bf6a7d06a6f25b682f8e/src/lua/digest.lua#L251).